app/uk/gov/hmrc/teamsandrepositories/persitence/model/BuildJob.scala

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmrc.teamsandrepositories.persitence.model
+
+import play.api.libs.json._
+
+case class BuildJob(service: String, jenkinsURL: String)
+
+object BuildJob {
+  implicit val formats: OFormat[BuildJob] =
```

Comment:
Effectively two implicit Writes[BuildJob] exist here - which will be picked up will depend on if functions are defined against `Writes[BuildJob]` or `Format[BuildJob]`
Is the `formats` defined for Mongo? If so, rename appropriately, and probably not make either implicit - but bring into scope as required.